### ISSUE TITLE
Feature: Switch added

### ISSUE THREAD

**Issue by carlsendk** [lukasroegner/homebridge-nello#20]
HomeKit: Switch to enable "auto open" if it switched on and you can use it to make intervals for opening the door

**Comment by AlexanderBabel** [comment]
Hey there,

during the upcoming weekend I will implement your request. But it not a promise that it will be finished at the end of the weekend.

Greetings
Alex

**Comment by carlsendk** [comment]
I don't know if it's the right way to implement it :-)

**Comment by AlexanderBabel** [comment]
Hey there,

I updated the plugin and added the switch and a motion sensor. Feel free to try it:

```bash
npm i git+https://github.com/lukasroegner/homebridge-nello.git#develop
```

Greetings
Alex

**Comment by AlexanderBabel** [comment]
A documentation of the new config entries can be found here: https://github.com/lukasroegner/homebridge-nello/tree/develop

**Comment by AlexanderBabel** [comment]
Available with v0.4.0
